page-impact.hbs

```diff
@@ -0,0 +1,550 @@
+{{!< default}}
+
+<div class="gh-page">
+    <div class="gh-container">
+
+        <!-- IMPACT-OUR-IMPACT -->
+        <div class="row">
+            <div class="col-xs-12 col-md-12 col-lg-12 mb-5">
+                <h1>Our Impact</h1>
+            </div>
+            <div class="col-xs-12 col-md-8 col-lg-8">
+                <article class="gh-card {{post_class}} {{#if feature_image}}with-image{{else}}no-image{{/if}}">
+                    <a class="gh-card-link" href="{{url}}">
+                        <div class="row no-gutters border rounded overflow-hidden flex-md-row align-items-center mb-4 shadow-sm h-md-250 position-relative">
+                            {{#if feature_image}}
+                            <div class="col-lg d-block">
+                                <img class="gh-card-image"
+                                    srcset="{{img_url feature_image size="s"}} 300w,
+                                            {{img_url feature_image size="m"}} 600w,
+                                            {{img_url feature_image size="l"}} 1000w,
+                                            {{img_url feature_image size="xl"}} 2000w"
+                                    sizes="(max-width: 1000px) 400px, 700px"
+                                    src="{{img_url feature_image size="m"}}"
+                                    alt="{{title}}"
+                                />
+                            </div>
+                            {{/if}}
+                            <div class="col-lg p-4 d-flex flex-column position-static gh-card-content">
+                                <!--<span class="gh-card-meta">
```

Comment:
Do delete the HTML comments for L29-32 and L35, since this should not depend on posts at all.